sys-boot/gnu_efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,31 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="
+Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), and IA-32 (x86)
+platforms using the GNU toolchain and the EFI development environment.
```

Comment:
The [guidelines]'https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines) recommend using this layout:
```
DESCRIPTION="Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), \
and IA-32 (x86) platforms using the GNU toolchain and the EFI development \
environment."
```